extensions/time.py

```
@@ -1,3 +1,11 @@
+#
+#   HANDLER
+#
+
+# HANDLER -> LOAD
+from website import *
```

Ruff (0.12.2)

`from website import *` used; unable to detect undefined names

(F403)